src/index.ts

```diff
@@ -32,7 +33,16 @@ const argv = yargs
 
 let cssLink = '';
 if (argv.stylesheet !== '') {
-    cssLink = argv.stylesheet;
+    if (isURL(argv.stylesheet)) {
+        cssLink = argv.stylesheet;
+    }
+    else {
```

Comment:
please move the `else` statement with a space to line 38 to comply with the existing code format.